Samples/HelloWorld.NetCore/Readme.md

```diff
@@ -0,0 +1,30 @@
+# HelloWorld.NetCore
+Orleans HelloWorld sample targeting .NET Core
+
+## Building the Sample
+If you are trying to use a version of Orleans that is not yet released, first run `Build.cmd netstandard` from the root directory of the repository.
+
+Note: If you need to reinstall packages (for example, after making changes to the Orleans runtime and rebuilding the packages), just manually delete all Orleans packages from `(rootfolder)/Samples/{specific-sample}/packages/` and re-run `Build.cmd netstandard`. You might sometimes also need to clean up the NuGet cache folder.
+
+You can then compile as usual, build solution.
+
+```
+dotnet restore
+dotnet build
+dotnet publish
+```
+
+## Running the Sample
+It's important to publish the apps before running them, as Orleans loads several assemblies via reflection and they need to be colocated.
```

Comment:
I don't know at this point. I think the current issue came from OrleansProviders, as it was not referenced, but for some reason the client needed it... I need to investigate a little bit more. In the meantime, I don't think we should hold off from merging the sample itself because of an issue with Orleans. We can iterate more, and it's easier for others if this is in the main repo.